bigbluebutton-html5/imports/api/group-chat/server/publishers.js

```diff
@@ -17,14 +16,10 @@ function groupChat(credentials) {
 
   return GroupChat.find({
     $or: [
-      {
-        access: CHAT_ACCESS_PUBLIC,
-        meetingId,
-      }, {
-        users: { $in: [requesterUserId] },
-        meetingId,
-      },
+      { meetingId },
```

Comment:
Won't this result in the publishing the information for every group chat in the meeting to every user? Seems like the other side of the OR is redundant.

Reply:
Yes, already changed it, it'll be resolved on next commit